packages/angular_devkit/build_angular/src/utils/normalize-file-replacements.ts

```diff
@@ -48,26 +49,26 @@ export function normalizeFileReplacements(
 
 function normalizeFileReplacement(
   fileReplacement: FileReplacement,
-  root?: Path,
+  root: string,
 ): NormalizedFileReplacement {
-  let replacePath: Path;
-  let withPath: Path;
+  let replacePath: string;
+  let withPath: string;
   if (fileReplacement.src && fileReplacement.replaceWith) {
-    replacePath = normalize(fileReplacement.src);
-    withPath = normalize(fileReplacement.replaceWith);
+    replacePath = path.normalize(fileReplacement.src);
+    withPath = path.normalize(fileReplacement.replaceWith);
   } else if (fileReplacement.replace && fileReplacement.with) {
-    replacePath = normalize(fileReplacement.replace);
-    withPath = normalize(fileReplacement.with);
+    replacePath = path.normalize(fileReplacement.replace);
```

Comment:
Do we need path normalization here? Especially considered that the segments are than joined using `path.join` which does normalization as well.